gigascience/gigadb-website
# Change button label and add tooltips on user profile tab for uploaded datasets

# Pull request for issue: #1613

This is a pull request for the following functionalities:

* Update operation button labels
* Add descriptive text and tooltips

![image](https://github.com/gigascience/gigadb-website/assets/143437854/6d4eea76-0b3e-49d7-929b-3a4ea73ab4d0)
![image](https://github.com/gigascience/gigadb-website/assets/143437854/d4f86cbb-6cd8-4c7e-b533-a3f143e69210)


## How to test?

* Navigate to this page as user which has one dataset in a pending state http://gigadb.gigasciencejournal.com/user/view_profile#submitted
* verify new text and new button labels
* hover over buttons and /or focus them with the keyboard and verify the tooltips

## How have functionalities been implemented?

* Button labels have been updated to specify if they refer to files or to the dataset
* Added a bootstrap tooltip to each button and styled them to look similar to the Unify tooltip (https://htmlstream.com/unify/documentation/tooltips.html)
* Tooltips are in a tenuous situation accessibility-wise, the general advice is to avoid them if the info they provide can be always visible, so I added the tooltips but also I decided to add helper information always visible before the table

## Any issues with implementation?

--

## Any changes to automated tests?

Two acceptance tests failing as described in this comment: https://github.com/gigascience/gigadb-website/pull/1642#issuecomment-1945567809

## Any changes to documentation?

--

## Any technical debt repayment?

--

## Any improvements to CI/CD pipeline?

--
